testproject/Assets/Prefabs/Cube.prefab

```diff
@@ -328,3 +329,15 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 9548116c10df1486ea12b7329b77c5cf, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!114 &2690316626396496521
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 8685790303553767886}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 962d0654df408407d8055453c9020f2b, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
```

Comment:
@jeffreyrainy — I think we probably shouldn't have merged this change (this is `ManualNetworkedVarTest` component being attached to `Cube.prefab` here which is not in testproject on `develop` branch)

I'll revert this change in one of my PRs next week.